lib/game_validator.rb

```diff
@@ -0,0 +1,78 @@
+require 'yaml'
+
+class GameValidator
+  def initialize(display, game_loader)
+    @display = display
+    @game_loader = game_loader
+  end
+
+  def validate_game_type_input(filename)
+    if existing_games_empty?(filename)
```

Comment:
You could use a tenary operator to make it shorter.

Reply:
Good point, thank you